### PR TITLE
(WIP) Test cases

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

**Comment by hasnain-naveed** [comment]
For management command test cases.
`./manage.py test edx_salesforce/management/tests --settings=settings.local`

For edx_data test cases.
`./manage.py test --settings=settings.local`

**Comment by hasnain-naveed** [comment]
```python
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.mysql',
        'NAME': 'edxapp',
        'USER': 'root',
        'PASSWORD': '',
        'HOST': '127.0.0.1',
        'PORT': '3306',
        'ATOMIC_REQUESTS': True,
        'TEST': {
            'ENGINE': 'django.db.backends.mysql',
            'NAME': 'test_edx',
            'USER': 'root',
            'PASSWORD': '',
            'HOST': '127.0.0.1',
            'PORT': '3306',
            'ATOMIC_REQUESTS': True,
        }
    },
    'ecommerce': {
        'ENGINE': 'django.db.backends.mysql',
        'NAME': 'ecommerce',
        'USER': 'root',
        'PASSWORD': '',
        'HOST': '127.0.0.1',
        'PORT': '3306',
        'ATOMIC_REQUESTS': True,
        'TEST': {
            'ENGINE': 'django.db.backends.mysql',
            'NAME': 'test_ecommerce',
            'USER': 'root',
            'PASSWORD': '',
            'HOST': '127.0.0.1',
            'PORT': '3306',
            'ATOMIC_REQUESTS': True,
        }
    }
}

File: edx_salesforce/management/commands/run_user_account_report.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def handle(self, *args, **options):
         )
 
         with open(output_filename, 'wb') as csvfile:  # pylint: disable=open-builtin
-            writer = csv.writer(csvfile, delimiter=',')
+            writer = csv.writer(csvfile, delimiter=str(','))
```
Comment (douglashall) [inline edx_salesforce/management/commands/run_user_account_report.py:102]: Do we need to wrap this in `str()`?

File: edx_salesforce/edx_data.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
         JOIN ecommerce_user AS u
         ON o.user_id = u.id
         WHERE
-        p.course_id REGEXP "^course-v1:({orgs}).*$"
+        p.course_id REGEXP "^course-v1:{orgs}.*$"
```
Comment (douglashall) [inline edx_salesforce/edx_data.py]: Why remove the `()`?

File: edx_salesforce/management/commands/run_user_account_report.py (hunk 1 above)
Comment (hasnain-naveed) [reply edx_salesforce/management/commands/run_user_account_report.py:102]: I was getting an error on that !!!
TypeError: "delimiter" must be string, not unicode

File: edx_salesforce/edx_data.py (hunk 2 above)
Comment (hasnain-naveed) [inline edx_salesforce/edx_data.py]: it is typo.

Comment (hasnain-naveed) [reply edx_salesforce/edx_data.py]: it was typo.

**Pull request merged**